src/components/MoviesListItem/MoviesListItem.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './MoviesListItem.scss';
+
+const MoviesListItem = (props) => {
+    const {
+        movieData,
+        raiseClickEvent,
+    } = props;
+
+    function handleClick() {
+        raiseClickEvent(movieData.id);
+    }
+
+    return (
+        // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+        <li className="movie-preview" onClick={handleClick} onKeyPress={handleClick}>
+            <img src={movieData.poster_path} alt="" className="movie-preview-poster"/>
+            <div className="movie-preview-header">
+                <p>{movieData.title}</p>
```

Comment:
how about heading here and paragraph below?

Reply:
Will be in future improvements.